tests/admin_views/tests.py

```diff
@@ -3156,6 +3156,22 @@ def test_model_admin_default_delete_action_no_change_url(self):
         # The page shouldn't display a link to the nonexistent change page
         self.assertContains(response, "<li>Unchangeable object: UnchangeableObject object</li>", 1, html=True)
 
+    def test_post_delete_action_protected(self):
+        """
+        A POST request to delete protected objects should display the page
+        which says the deletion is prohibited.
+        """
+        q = Question.objects.create(question='Why?')
+        Answer.objects.create(question=q, answer='Because.')
+
+        response = self.client.post(reverse('admin:admin_views_question_changelist'), {
```

Comment:
I think you could add this to `test_model_admin_default_delete_action_protected` so you don't have to repeat the setup steps.